### PR TITLE
lots of changes

### PR DESCRIPTION
**Pull request by Huskydog9988**

### CODE REVIEW

File: charts/website/templates/deployment.yaml [hunk 1]
```diff
@@ -33,6 +33,130 @@ spec:
...
+                  key:  client-secret
+                  name: {{ .Release.Name }}-facebook
+
+            # cloudflare
+            - name: "CLOUDFLARE_TURNSTILE_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  turnstile-secret
+                  name: {{ .Release.Name }}-cloudflare
+            - name: "CLOUDFLARE_TURNSTILE_SITE_KEY"
```
Comment (finnc0) [inline charts/website/templates/deployment.yaml:123]: Needs to be changed to NEXT_PUBLIC_CLOUDFLARE_TURNSTILE_SITE_KEY

File: charts/website/templates/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  client-id: {{ .Values.siteConfig.facebook.client-id | b64enc }}
+  client-secret: {{ .Values.siteConfig.facebook.client-secret | b64enc }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-cloudflare
+data:
+  turnstile-secret: {{ .Values.siteConfig.cloudflare.turnstile-secret | b64enc }}
+  turnstile-site-key: {{ .Values.siteConfig.cloudflare.turnstile-site-key | b64enc }}
```
Comment (finnc0) [inline charts/website/templates/secret.yaml:56]: add next_public here 2 ^ (if needed)

File: charts/website/templates/deployment.yaml (hunk 1 above)
Comment (Huskydog9988) [reply charts/website/templates/deployment.yaml:123]: k, i'll do that in the next release, this is still using 0.0.1-alpha.2 which doesn't have that

Comment (finnc0) [reply charts/website/templates/deployment.yaml:123]: kk

**Review by finnc0 — APPROVED** [review]

**Review by tsundDEV — APPROVED** [review]

File: charts/website/templates/secret.yaml (hunk 2 above)
Comment (Huskydog9988) [reply charts/website/templates/secret.yaml:56]: turnstile secret is public?

Comment (Huskydog9988) [reply charts/website/templates/secret.yaml:56]: resolved in discord, only site key needs to be

**Comment by Huskydog9988** [comment]
github must be broken today, its already merged in ded435c

**Pull request closed without merging**